lib/harmony/harmonydb/sql/20231110.sql

```diff
@@ -0,0 +1,39 @@
+create table mining_tasks
```

Comment:
Should we maybe also persist info about which instance executed the task?  🤔 That might be helpful in analyzing and optimising multi-worker environment

Reply:
We have the completed tasks table for harmonytask. We could add the field there and update it for all completed tasks.